dev/ci/README-users.md

```diff
@@ -45,6 +45,20 @@ using [coqbot](https://github.com/coq/bot); meanwhile, a workaround is
 to give merge permissions to someone from the Coq team as to help with
 these kind of merges.
 
+### OCaml and plugin-specific considerations
+
+Developments that link against Coq's OCaml API [most of them are known
+as "plugins"] do have some special requirements:
+
+- Coq's OCaml API is not stable. We hope to improve this in the future
+  but as of today you should expect to have to merge a large amount of
+  overlays. You can use some automated tool such as
+  [Bors](https://github.com/apps/bors) to help Coq developers.
+
+- Plugins in the CI should compile with the same OCaml flags than Coq
+  itself. In particular, warnings that are considered errors _must_
+  not be disabled by plugin authors.
```

Comment:
>In particular, warnings that are considered errors _must_
  not be disabled by plugin authors.

Not sure how clear that is to someone who doesn't already know what it means. OTOH I don't know what would be better.